src/fwd_analyzer.cpp

```
@@ -55,9 +54,9 @@ class InterleavedFwdFixpointIterator final {
     void set_pre(const Label& label, EbpfDomain&& v) { result.invariants.at(label).pre = std::move(v); }
     void set_pre(const Label& label, const EbpfDomain& v) { result.invariants.at(label).pre = v; }
 
-    EbpfDomain get_pre(const Label& node) const { return result.invariants.at(node).pre; }
+    const EbpfDomain& get_pre(const Label& node) const { return result.invariants.at(node).pre; }
```

Clang (14.0.6)

[warning] function 'get_pre' should be marked [[nodiscard]]

(modernize-use-nodiscard)